DWM/update_abos_attributes_to_dwm.py

```diff
@@ -38,34 +37,44 @@ def get_netcdf_files(folder: str):
     """ Get the nc files in the path given when calling the script in the command line """
     return glob(os.path.join(folder, '**/*.nc'), recursive=True)
 
- 
+
 def fix_abos_files(folder: str):
     """ Call the functions to replace attributes that contain 'ABOS' and/or 'Australia Bluewater Observing System') """
     files = get_netcdf_files(folder)
-
     for file in files:
         ncobj = nc.Dataset(file, 'a')
```

Comment:
Add a log line here. Something like "Updating file {file}"

Reply:
I think this is all fixed now

Reply:
:+1: